Source/Form/Gen3/Static3.cpp

```diff
@@ -153,14 +153,15 @@ void Static3::generatorCategoryIndexChanged(int index)
                     QVariant::fromValue(i));
             }
         }
+        ui->comboBoxGeneratorMethod->setItemHidden(1, templates[ui->comboBoxGeneratorPokemon->getCurrentInt()].getBuggedRoamer());
```

Comment:
What is this for?

Reply:
It removes method 4 when a bugged roamer is selected. Method 4 never happens, cause the rng call for ivs2 is not considered at all